### ISSUE TITLE
Fieldset and legends - should go to globals

### ISSUE THREAD

**Issue by Januszpl** [SnowdogApps/magento2-alpaca-components#246]
Currently they are in elements (I've moved them in the beginning when we removed form module).

We have few option:

1. use classes fieldset and fieldset__legend everytime we want alpaca default styles for those elements
2. separate and move it to globals and style default legend and fieldset tag  with classes ".legend" and ".fieldset" - I would choose this one as we can easily build forms in place where they will be and adding any custom html like in example below won't be a problem (where at 3rd option this can cause a problem)
3.  Use `{{ render (component content) contentContext }}` in the loop in current element ? Perhaps with some `beforeFieldHtml` and `afterFieldHtml` properties in config js so we can render any custom html structure (but Don't like this idea as I think sometimes this html should be in .hbs file not in .config.js)

However if we will create form ex.
```
           <fieldset>
               <legend class="registration__legend {{ signInSection.legend.class }}"
                       {{{ firstSection.legend.attributes }}}
               >
                </legend>

                {{ render '@input' email }}

                {{ render '@input' password }}
                {{ render '@password-strength' }}

                {{ render '@input' passwordConfirm }}
                
                <div class="registration__marketing-content">
                   {{ someText }}
                </div>

            </fieldset>
```

**Comment by kruchy8** [comment]
Include/extend `.fieldset` class in `fieldset` tag like with headers
```
h1,
h2,
h3,
h4,
h5,
h6 {
    @extend .heading;
}
```

**Comment by Januszpl** [comment]
Yes. it's actually what I meant by option 2. so move it to globals and add as you said
```
.fieldset {
...
}
fieldset {
    @extend .fieldset
}
```

**Comment by Januszpl** [comment]
@anqaka what do you think about it ?
